README.md

```diff
@@ -1,10 +1,16 @@
+<center>
+
+![esp-rs Logo](./resources/esp-rs.svg)
```

Comment:
Oooo fancy!

Reply:
Unfortunately it doesn't seem to be rendering correctly on GitHub (for me at least) 😅

Reply:
https://github.com/jessebraham/esp-hal/blob/fixes/misc/README.md it renders for me? Unless you're talking about the position?

Reply:
It's not centered, I guess GitHub doesn't allow HTML tags (which is in violation of the Markdown spec)

Reply:
Maybe something like this works?

https://github.com/waldyr/sublime-installer/blob/07c5919394858f91e60dc5c7f095da9b3165a32c/README.md?plain=1#L39-L42